tests/test-qemu.expected.log

```diff
@@ -1,6 +1,8 @@
 
 # insmod keystone-driver.ko
-# ./test
+# cd tests; ./tests.ke
+Verifying archive integrity... All good.
+Uncompressing Keystone vault archive     0%    47%    95%  100%  

```

Comment:
Is this consistent with the %s? Ideally we'd not check this line since I expect it will change with file size.

Reply:
okay. that'd require few lines for in the script.